packages/radix-vue/src/Editable/EditableRoot.vue

```diff
@@ -199,6 +204,9 @@ provideEditableRootContext({
         :model-value="modelValue"
         :is-editing="isEditing"
         :is-empty="isEmpty"
+        :submit="submit"
+        :cancel="cancel"
+        :edit="edit"
```

Comment:
let's expose these function in `defineExpose` too 😁

Reply:
Functions added to `defineExpose` 👍🏻